src/olympia/users/tests/test_admin.py

```diff
@@ -93,35 +93,70 @@ def test_search_ip_as_int_isnt_considered_an_ip(self):
         assert response.status_code == 200
         doc = pq(response.content)
         assert not doc('#result_list tbody tr')
-        assert not doc('.column-_ratings_all__ip_address')
+        assert not doc('.column-known_ip_adresses')
 
     def test_search_for_single_ip(self):
         user = user_factory(email='someone@mozilla.com')
         self.grant_permission(user, 'Users:Edit')
         self.client.force_login(user)
-        user_factory(last_login_ip='127.0.0.1')  # Extra user that shouldn't match
-        self.user.update(email='foo@bar.com', last_login_ip='127.0.0.2')  # Will match
+        with core.override_remote_addr('127.0.0.2'):
```

Comment:
can we have some `assertNumQueries` in these tests?  It'd be good to know how we're doing currently, at least.

Reply:
https://github.com/mozilla/addons-server/pull/19626/commits/39ce33266b5430775385dc78e02d756e62a722e1